pandas/tests/scalar/test_timestamp.py

```diff
@@ -704,6 +471,31 @@ def test_today(self):
         assert (abs(ts_from_string_tz.tz_localize(None) -
                     ts_from_method_tz.tz_localize(None)) < delta)
 
+
+class TestTimestamp(object):
+
```

Comment:
should go in scalar/test_timezones , yes?

Reply:
I decided to leave this one out of this round until I make up my mind on exactly where it belongs.  `test_tz` isn't very informative, so it will likely be split into method-specific tests, many of which likely already exist.